src/ThirdEye.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<SPDXLicenseCollection xmlns="http://www.spdx.org/license">
+   <license isOsiApproved="false" licenseId="ThirdEye"
+   name="ThirdEye License" listVersionAdded="">
+      <crossRefs>
+         <crossRef>https://sourceware.org/cgit/binutils-gdb/tree/include/coff/symconst.h#n11</crossRef>
+      </crossRefs>
+      <notes>
+         This is a variant to MIPS license with extra paragraph.
+      </notes>
+      <text>
+         <p>
+            (C) Copyright 1984 by Third Eye Software, Inc.
+
+            Third Eye Software, Inc. grants reproduction and use rights to
+            all parties, PROVIDED that this comment is maintained in the copy.
+
+            Third Eye makes no claims about the applicability of this
+            symbol table to a particular use.
+         </p>
```

Comment:
I'd suggest breaking these up into separate `<p>` sections for each one, since in the original and test text they show up as separate paragraphs.